src/ci/utils.py

```diff
@@ -344,6 +407,40 @@ def runASAN(moduleName):
 
     printGreen("<"+moduleName+">"+"[ASAN: PASSED]"+"<"+moduleName+">")
 
+def runScanBuild(targetName):
+    """
+    Executes scan-build for 'targetName'.
+
+    :param targetName: Target to be analyzed using scan-build analysis tool.
+    """
+    printHeader("<"+targetName+">"+headerDic['scanbuild']+"<"+targetName+">")
+    cleanAll()
+    cleanExternals()
+    makeDeps(targetName)
```

Comment:
For Windows, you should use the new argument added (`make deps EXTERNAL_SRC_ONLY=yes`) to get non-compiled libraries.